lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'code',
+    schema: [{
+      type: 'object',
+      properties: {
+        'firstline': {
+          type: 'number',
+          minimum: 0
+        },
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'number',
+          minimum: 1
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0]
+    const multilineMaximum = configuration.multiline
+    const singlelinemMaximum = configuration.singleline
+    const canHaveFirstLine = configuration.firstline !== 0
```

Comment:
I think `boolean` or `string` option is more proper in this case.
How about the below?

**A:** Simple boolean option.

```js
{
    "singleline": 3,
    "multiline": 1,
    "allowFirstLine": false // or true
}
```

**B:** I think that the first attribute position is a taste for each people. It might be good if people can specify the position. But I'm not sure it's a responsibility of this rule.

```js
{
    "singleline": 3,
    "multiline": 1,
    "firstAttributeOfMultiline": "newline" // or "beside"
}
```

**C:** I'm wondering that it's a setting of multiline. Should it locate in the multiline?

```js
{
    "singleline": 3,
    "multiline": {"max": 1, "allowFirstLine": false}
}
```